src/cli/editor.py

```
@@ -61,7 +62,7 @@ def get_editor_input(content: str | None= None) -> str:
             tf.write(content.encode())
             tf.flush()
         try:
-            subprocess.call([editor, tf.name])
+            subprocess.call(shlex.split(editor) + [tf.name])
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)